### ISSUE TITLE
[BUG] test.info().attach is showing same content

### ISSUE THREAD

**Issue by xyngfei** [cenfun/monocart-reporter#62]
Let's say I have 2 tests, and run both tests concurrently. 
However, both tests have shown the same test attach content, need your help to looking into it

```
test("@GCC-7777|T2", async ({}) => {
    test.setTimeout(10 * 1000);

    test.info().attach("application/json stringify:", {
      contentType: "application/json",
      body: JSON.stringify(
        {
          Name: "1",
          Text: "2",
          Text2: "3",
          Text3: "4",
        },
        null,
        4
      ),
    });
  });

  test("@GCC-7777|T3", async ({}) => {
    test.setTimeout(10 * 1000);

    test.info().attach("application/json stringify:", {
      contentType: "application/json",
      body: JSON.stringify(
        {
          Name: "9",
          Text: "8",
          Text2: "7",
          Text3: "6",
        },
        null,
        4
      ),
    });
  });
```

**Comment by cenfun** [comment]
it should be fixed in monocart-reporter@1.7.11

**Comment by xyngfei** [comment]
Yes, it works as expected. thx again for speedy resolution
